Bio/SeqIO/FastaIO.py

```diff
@@ -140,7 +143,12 @@ def FastaTwoLineParser(handle):
 class FastaIterator(SequenceIterator):
     """Parser for Fasta files."""
 
-    def __init__(self, source, alphabet=None, title2ids=None):
+    def __init__(
+        self,
+        source: Union[str, TextIO],
+        alphabet=None,
+        title2ids: Optional[Title2IDsFuncType] = None,
```

Comment:
The ``title2ids`` argument is deprecated and due to be removed soon - can we not bother typing it to avoid introducing ``Title2IDsFunctType`` and then removing it?

Reply:
Makes sense. I've removed the annotation.

Reply:
I didn't realise initially this was based on the branch from 2 years ago.